packages/buefy-next/src/components/upload/Upload.vue

```diff
@@ -71,44 +80,35 @@ export default {
         }
     },
     emits: ['invalid', 'update:modelValue'],
-    data() {
+    data(): BUploadData {
```

Comment:
I would simply let TypeScript infer the type of `data`; i.e., would remove `: BUploadData` here.